hibernate-reactive-core/src/test/java/org/hibernate/reactive/EagerManyToOneAssociationTest.java

```diff
@@ -101,7 +101,7 @@ public void manyToOneIsNull(TestContext context) {
 	@DiscriminatorValue("N")
 	//@Inheritance(strategy = InheritanceType.JOINED)
 	public static class Book {
-		public static final String TABLE = "Book";
+		public static final String TABLE = "Bookz";
```

Comment:
Just a curiosity, how come you also need to change the table name?

Reply:
Well MariaDB seems to be a lot pickier about that stuff for some reason I wasn't able to fully nail down. All I know is I wish it supported `cascade`.

Reply:
(In `drop table` I mean.)